### ISSUE TITLE
Document archiving a dataset

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/hca-ebi-wrangler-central#62]
**Description of the task:**

Document archiving a dataset from scratch. The steps will include:

1. Creating a submission
1. Validating metadata
1. Uploading data files
1. Submitting to ingest
1. Running the archiver for the metadata.
1. Running the archiver for data
1. Correcting metadata validation errors in DSP

This [document](https://docs.google.com/document/d/1FfKxrBqZxPB6RyaBDMkyw5lmucH5FpBDnoiu2B6wJyk/edit) will showcase all the steps followed in detail.

The purpose of this ticket is to identify unexpected issues, bottlenecks and document an example for archiving. Might be extended to cover DCP1 datasets.

**Acceptance criteria for the task:**

- [x] The process has been documented thoroughly
- [ ] The data and metadata for https://github.com/HumanCellAtlas/hca-data-wrangling/issues/115 displays correctly in the archives.

**Comment by ESapenaVentura** [comment]
Finished the document, waiting on the file archiver to finish the last job and we will be able to close this ticket!

**Comment by ESapenaVentura** [comment]
The data is not yet ready due to a problem with DSP's file validation but the documentation is already here.

This is a practical but most of the steps are not needed anymore. I documented this just in case we still needed workarounds but experience with other projects tells me the process is well documented in the original documentation.

Closing the ticket and posting updates in the tracker ticket

Closing the ticket and will k
